synth.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import synthtool as s
+import synthtool.gcp as gcp
+import logging
+
+logging.basicConfig(level=logging.DEBUG)
+common_templates = gcp.CommonTemplates()
+templates = common_templates.node_library()
+s.copy(templates, excludes=['README.md'])
+
+s.replace('.kokoro/test.bat', r'(call npm install \|\| goto :error)',
```

Comment:
Huh - what's up with that?  Would be nice to have expansive comments through here describing why we're overriding the behavior for each one.

Reply:
Hmm, I figured the inline replacement comments would explain what is happening. But I'll comments in this file as well.

Reply:
You are right, my brain was not on correctly this morning.